app/serializers/post_serializer.rb

```diff
@@ -338,6 +343,10 @@ def bookmark_reminder_at
     post_bookmark&.reminder_at
   end
 
+  def bookmark_reminder_type
+    Bookmark.reminder_types[post_bookmark&.reminder_type].to_s
```

Comment:
This is a bit odd - do we want to look up `nil` in a hash then call `to_s` on the resulting `nil`? Should be an empty string but it seems weird to me.